integration/services/ha_backend/setup.sh

```diff
@@ -145,5 +143,4 @@ HAB_LICENSE="accept-no-persist" hab svc load ${postgresql_pkg_ident} --topology
 HAB_LICENSE="accept-no-persist" hab svc load ${pgleaderchk_pkg_ident} --topology leader --bind database:"$PG_PKG_NAME".default --binding-mode=relaxed --channel ${channel}
 HAB_LICENSE="accept-no-persist" hab svc load ${proxy_pkg_ident} --topology leader --bind database:"$PG_PKG_NAME".default --bind pgleaderchk:"$PGLEADERCHK_PKG_NAME".default --binding-mode=relaxed --channel ${channel}
 HAB_LICENSE="accept-no-persist" hab svc load ${elasticsearch_pkg_ident} --topology leader --channel ${channel}
-# TODO: Enabled elasticsidecar when a build of it lands in the depot
-# HAB_LICENSE="accept-no-persist" hab svc load ${elasticsidecar_pkg_ident} --topology leader --bind elasticsearch:"$ELASTICSEARCH_PKG_NAME".default --binding-mode=relaxed --channel ${channel}
+HAB_LICENSE="accept-no-persist" hab svc load ${elasticsidecar_pkg_ident} --topology leader --bind elasticsearch:"$ELASTICSEARCH_PKG_NAME".default --binding-mode=relaxed --channel ${channel}
```

Comment:
Just wanted to point out that only PostgreSQL is in a leader topology, the rest are `standalone`
At one point, I think `leader` caused issues with Elasticsearch. Anyways, the only service that truly needs leader is PG.

https://github.com/chef/a2-ha-backend/blob/9a75983bcc6b432660bdd2017d0df9ce7b8e9a40/terraform/variables_common.tf#L75-L89